### PR TITLE
Make the cursor movements default to 1 step if not provided - Fixes #39

### LINKED ISSUE #39: Cursor movement commands don't default to 1
According to [my reading](http://www.termsys.demon.co.uk/vtansi.htm) a cursor movment such as `\x1b[A` should be interpreted as `\x1b[1A` so the default movement is 1 character.  This is obeyed by xterms I've tested.

However `\x1b[A` is ignored by go-colorable because of this code

https://github.com/mattn/go-colorable/blob/72ec6f98a4a16d5106d74ec2fa11b14d6c5a28b3/colorable_windows.go#L502-L506

I think this problem affects `A`, `B`, `C`, `D`

Let me know if you would like me to send a patch to fix this.

### PR DESCRIPTION
**Pull request by ncw**

### CODE REVIEW

File: colorable_windows.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	"syscall"
 	"unsafe"
 
-	"github.com/mattn/go-isatty"
+	isatty "github.com/mattn/go-isatty"
```
Comment (mattn) [inline colorable_windows.go]: Please revert this line.

**Comment by mattn** [comment]
https://github.com/mattn/go-colorable/pull/42 is required ?

File: colorable_windows.go (hunk 1 above)
Comment (ncw) [reply colorable_windows.go]: I've done that now!  Sorry that is `goimports` which I use on a save-hook

**Comment by ncw** [comment]
> #42 is required ?

It would be good for compatibility I think.

**Comment by mattn** [comment]
Thanks

**Pull request merged**